src/Dfe.EarlyYearsQualification.Web/wwwroot/js/site.js

```diff
@@ -2,3 +2,12 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+function printButtonClicked()

```

Comment:
Accessibility question (not even necessarily for this PR): I have just noticed that the print button only has an `onClicked` event handler -- does that work for keyboard navigation too, or does this function need to be invoked from the button's `onKeyPress` also?

Reply:
I just checked running this locally and tabbing to the button and pressing space or enter clicks the button :-)